Jenkinsfile

```diff
@@ -58,6 +59,32 @@ void unittests(String version) {
     }
 }
 
+void uploadPackagePyPI() {
+    node(Constants.DOCKERNODE) {
+        buildsCommon.cleanup()
+        checkout scm
+        withContainer(image: "python:3.6-buster", registry: '', inside: '-u root') {
+            try {
+                if ((env.BRANCH_NAME == 'master') && (releaseBuild == 'Yes')) {
```

Comment:
Why here? You start up an EC2 checkout source code, start a container to then decide not to do anything????